### PR TITLE
P2022-725 Lack of internet connection -> P2022-949 Check connection

### PR DESCRIPTION
**Pull request by kmaslowiec**

### CODE REVIEW

**Review by catchanus — APPROVED** [review]

File: build.gradle [hunk 1]
```diff
@@ -8,7 +8,7 @@ buildscript {
         }
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:7.0.4"
+        classpath 'com.android.tools.build:gradle:7.1.0'
```
Comment (MarekMacko) [inline build.gradle:11]: Update of Android Gradle Plugin should be done as a separate PR

File: app/src/main/java/com/intive/patronage/retro/MainActivity.kt [hunk 2]
```diff
@@ -82,4 +86,15 @@ class MainActivity : AppCompatActivity() {
         super.onPause()
         binding.drawerLayout.visibility = View.GONE
     }
+
+    private fun callNetworkConnection() {
+        checkNet = CheckNetworkConnect(application)
```
Comment (MarekMacko) [inline app/src/main/java/com/intive/patronage/retro/MainActivity.kt:91]: Object instantiation should be a responsibility of our DI. In short: inject it :)

Comment (kmaslowiec) [reply app/src/main/java/com/intive/patronage/retro/MainActivity.kt]: Done :)

File: build.gradle (hunk 1 above)
Comment (kmaslowiec) [reply build.gradle]: I think I will do it in the next PR
